docs/se/reactivemessaging/01_introduction.adoc

```diff
@@ -20,6 +20,96 @@
+As the making of connectors or even configuring them can be repetitive task leading to
+the same results, Helidon SE Reactive Messaging supports very same configuration format
+for connectors as its MicroProfile counterpart does. Also, MP Connectors are reusable in
+Helidon SE Messaging with some limitation(there is no CDI in Helidon SE).
+All Messaging connectors in Helidon are made to be universally usable by Helidon MP and SE.
+
+=== Channel
+Channel is a named pair of `Publisher` and `Subscriber`, channels can be connected together by <<Processors,processors>>. Registering of `Publisher` or `Subscriber` for a channel can be done by Messaging API, or configured implicitly for using registered <<se/03_connector.adoc,connector>> for generating such `Publisher` or `Subscriber`.
+
+[source,java]
+.Example of simple channel:
+----
+Channel<String> channel1 = Channel.create("channel1");
+
+Messaging.builder()
+        .publisher(channel1, Multi.just("message 1", "message 2")
+                                  .map(Message::of))
+        .listener(channel1, s -> System.out.println("Intecepted message " + s))
+        .build()
+        .start();
+----
+
+=== Processor
+Processor is a typical reactive processor acting as a `Subscriber` to upstream and as a `Publisher` to downstream. In terms of reactive messaging it is able to connect two <<Channel,channels>> to one reactive stream.
+
+[source,java]
+.Example of processor usage:
+----
+Channel<String> firstChannel = Channel.create("first-channel");
+Channel<String> secondChannel = Channel.create("second-channel");
+
+Messaging.builder()
+        .publisher(secondChannel, ReactiveStreams.of("test1", "test2", "test3")
+                .map(Message::of))
+        .processor(secondChannel, firstChannel, ReactiveStreams.<Message<String>>builder()
+                .map(Message::getPayload)
+                .map(String::toUpperCase)
+                .map(Message::of)
+        )
+        .subscriber(firstChannel, ReactiveStreams.<Message<String>>builder()
+                .peek(Message::ack)
+                .map(Message::getPayload)
+                .forEach(s -> System.out.println("Consuming message " + s)))
+        .build()
+        .start();
+
+>Consuming message TEST1 
+>Consuming message TEST2
+>Consuming message TEST3
+----
+
+=== Message
+Reactive Messaging in Helidon SE is using same concept of <<mp/reactivemessaging/02_message.adoc,Message wrapper>> as Microprofile messaging, only notable difference is that SE Messaging does almost no implicit or automatic acknowledgement due to _no magic_ phylosophy of Helidon SE. 
+
+Only exception to this are variants of methods `Messaging.Builder#listener` and `Messaging.Builder#processor` with consumer or function params, conviniently unwraping payload for you. After such implicit unwraping is not possible to do manual acknowledgement, therefore implicit ack before callback is executed is necessary.   
```

Comment:
Reactive Messaging in Helidon SE uses the same concept of messaging wrapping as Microprofile messaging. The only notable difference is that ....

Reply:
thx fixed